static/app/components/group/sidebar.tsx

```diff
@@ -211,7 +211,7 @@ class BaseGroupSidebar extends Component<Props, State> {
 
         {this.renderPluginIssue()}
 
-        <SidebarSection title={t('Tags')}>
+        <SidebarSection title={t('Tag Summary')}>
```

Comment:
Making the copy consistent with how we describe this on the Transaction Summary page. We shouldn’t have two different names for these.